packages/ketchup/src/f-components/f-button/f-button.scss

```diff
@@ -1,4 +1,7 @@
 .f-button--wrapper {
+  --kup-button-spinner-width: 1.475em;
+  --kup-button-spinner-height: 1.475em;
```

Comment:
Replace --kup-button-spinner-width with --kup-button_spinner-width.
Same for height.